jest.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
 	globals: {
 		wcSettings: {
 			pinterest_for_woocommerce: {
+				pluginVersion: '1.2.3',
```

Comment:
❓ 

Why is this for?

Reply:
`LandinPageApp` requires the presence of `wcSettings.pinterest_for_woocommerce.pluginVersion` to run:
https://github.com/woocommerce/pinterest-for-woocommerce/blob/add/232-setup-event/assets/source/setup-guide/app/views/LandingPageApp.js#L271:L271